diff --git a/libgnucash/engine/gnc-commodity.c b/libgnucash/engine/gnc-commodity.c
index 79d988a212..f33dbf1cd0 100644
--- a/libgnucash/engine/gnc-commodity.c
+++ b/libgnucash/engine/gnc-commodity.c
@@ -236,6 +236,8 @@ static gnc_quote_source single_quote_sources[] =
     { FALSE, 0, 0, "VWD, DE (unmaintained)", "VWD", "vwd" },
     { FALSE, 0, 0, "Yahoo as JSON", "YAHOO_JSON", "yahoo_json" },
     { FALSE, 0, 0, "Yahoo as YQL", "YAHOO_YQL", "yahoo_yql" },
+    { FALSE, 0, 0, "Yahoo Web", "YAHOOWEB", "yahooweb" },
+    { FALSE, 0, 0, "Google Web", "GOOGLEWEB", "googleweb" },
 };
 
 static gnc_quote_source multiple_quote_sources[] =
