diff --git a/whatapi.py b/whatapi.py
index 66e0d3d..c3110fc 100644
--- a/whatapi.py
+++ b/whatapi.py
@@ -88,16 +88,11 @@ class WhatAPI:
         '''Logs in user and gets authkey from server'''
         loginpage = '{0}/login.php'.format(self.endpoint)
         data = {'username': self.username,
-                'password': self.password}
+                'password': self.password,
+                'twofa': self.totp}
         r = self.session.post(loginpage, data=data)
         if r.status_code != 200:
             raise LoginException
-        if self.totp:
-            params = {'act': '2fa'}
-            data = {'2fa': self.totp}
-            r = self.session.post(loginpage, params=params, data=data)
-            if r.status_code != 200:
-                raise LoginException
         accountinfo = self.request('index')
         self.authkey = accountinfo['authkey']
         self.passkey = accountinfo['passkey']
