diff -ru gnucash-4.12/gnucash/price-quotes.scm gnucash-4.12_/gnucash/price-quotes.scm
--- gnucash-4.12/gnucash/price-quotes.scm	2023-01-04 01:40:46.048093461 +0900
+++ gnucash-4.12_/gnucash/price-quotes.scm	2023-01-04 01:39:44.602261110 +0900
@@ -44,7 +44,7 @@
     (define (start-program)
       (set! program
         (gnc-spawn-process-async
-         (list "perl" "-w" gnc:*finance-quote-check*) #t)))
+         (list gnc:*finance-quote-check*) #t)))
 
     (define (get-sources)
       (when program
@@ -119,7 +119,7 @@
 
     (define (start-quoter)
       (set! quoter
-        (gnc-spawn-process-async (list "perl" "-w" gnc:*finance-quote-helper*) #t)))
+        (gnc-spawn-process-async (list gnc:*finance-quote-helper*) #t)))
 
     (define (get-quotes)
       (when quoter
